### PR TITLE
MAINT improved installation instructions

### LINKED ISSUE #242: RELEASE requirements?
Hi,

When doing:

```
conda install pytorch torchvision -c pytorch
pip install -i https://test.pypi.org/simple/ kymatio
```

then cloning the corresponding release, and doing `pytest`, I have tons of errors that are evidences that none of the requirements.txt packages are installed. Actually, this makes me think we should have a pip for the GPU version(like `pip install kymatio-gpu`)?

https://github.com/kymatio/kymatio/blob/master/requirements.txt

### PR DESCRIPTION
**Pull request by edouardoyallon**
Small modifications of the readme to make it more coherent with pip + improving the setup.py with more attributes to setup_tools (e.g. python_requires) and removing the support of "Windows" and python<3.6

Close #242

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # Constants
 DISTNAME = 'kymatio'
 DESCRIPTION = 'Wavelet scattering transforms in Python with GPU acceleration'
-URL = 'https://kymatio.github.io'
+URL = 'https://www.kymat.io'
```
Comment (lostanlen) [inline setup.py]: good catch :)

File: README.md [hunk 2]
```diff
@@ -64,23 +64,13 @@ pip install kymatio
...
-
-
 ### GPU acceleration
 
 
 To run Kymatio on a graphics processing unit (GPU), you should install the CUDA library and install the scikit-cuda dependency by running the following pip command:
 
 ```
-pip install -r requirements_optional_cuda.txt
+pip install -r https://raw.githubusercontent.com/kymatio/kymatio/master/requirements_optional.txt
```
Comment (lostanlen) [inline README.md]: why not 
`pip install -r https://raw.githubusercontent.com/kymatio/kymatio/master/requirements_optional_cuda.txt` ?
(note the `_cuda` suffix)

Comment (edouardoyallon) [reply README.md]: wow gosh, my bad

Comment (eickenberg) [reply README.md]: btw why are you linking to the file online and not the local one? Not sure that is common practice

Comment (edouardoyallon) [reply README.md]: What is the path without cloning it?!

File: setup.py [hunk 3]
```diff
@@ -47,15 +47,18 @@
                   'tomas.angles@ens.fr, gabriel.huang@umontreal.ca, roberto.leonarduzzi@ens.fr'),
     url=URL,
     download_url='https://github.com/kymatio/kymatio/releases',
+    project_urls={
+        'Documentation': 'https://www.kymat.io/codereference.html',
+        'Source': 'https://github.com/kymatio/kymatio/',
+        'Tracker': 'https://github.com/kymatio/kymatio/issues',
+        'Authors': 'https://github.com/kymatio/kymatio/blob/master/AUTHORS.md'
```
Comment (eickenberg) [inline setup.py]: Is this a link for humans or machines? If for machines, maybe raw link is better. If for humans, this link is better

Comment (edouardoyallon) [reply setup.py]: humen AFAIK

File: README.md (hunk 2 above)
Comment (lostanlen) [reply README.md]: another, really way to do it (for the time being) is
```
pip install scikit-cuda cupy
```

Comment (lostanlen) [reply README.md]: we have so few GPU-related dependencies for the moment that it might be overkill to write a `requirements.txt` file just for two lines

Comment (edouardoyallon) [reply README.md]: Solved

**Comment by edouardoyallon** [comment]
Please review and merge!

**Pull request merged**